build_r_site.R

```diff
@@ -0,0 +1,23 @@
+library(pkgdown)
+
+setwd("lightgbm_r")
+if (!dir.exists("docs")) {
+  dir.create("docs")
+}
+
+devtools::document()
+clean_site()
+init_site()
+build_home(preview = FALSE, quiet = FALSE)
+build_reference(lazy = FALSE, document = FALSE,
```

Comment:
It's quite strange, but removing `devtools::document()` and setting `document = TRUE` result in broken page:

![image](https://user-images.githubusercontent.com/25141164/57889066-7e122500-783c-11e9-81ba-6baf50ed1878.png)

https://github.com/r-lib/pkgdown/blob/96e12f83bb3f92e4a30309181cc14134bd92b644/R/build-reference.R#L105-L107

Reply:
@StrikerRUS Can you use `roxygen2::roxygenize("mypackagefolder", roclets = "rd")` instead of building the package with `devtools::document()` to update the Rd documentation?

Note that the rd documents you removed must be generated and still stored in LightGBM repository. They are essential for the R (offline) documentation and should not be generated by the user.

Reply:
@Laurae2 
> Note that the rd documents you removed must be generated and still stored in LightGBM repository. 

Thanks! I understand this clearly. At this moment this PR is in "Draft" stage and I removed all `Rd` files just to ensure that they can be generated on the RTD site, but not reused.

However, I'd like to ask someone else to regenerate these files (I have no R installed on my machine) right after this PR or I can place them back but they will be outdated because I modified R source file a little bit.